tests/run/dynamicDynamicTests.scala

```diff
@@ -1,3 +1,4 @@
+import dotty.language.dynamics
```

Comment:
Unless this works very differently from dynamic in scalac, I think we should use `import scala.language.dynamics` for compatibility